### DISCUSSION TITLE
请问Lsposed模式是还未支持安卓15吗

### DISCUSSION THREAD

**Discussion by TwooSix** [chinosk6/gakuen-imas-localify#48] [General]
hyperos2，用的[LSPosed_mod](https://github.com/mywalkb/LSPosed_mod) 这个仓库的lsposed，进游戏是白屏

**Comment by chinosk6** [comment]
试试[这个](https://github.com/JingMatrix/LSPosed)
有群友在这个版本的LSPatch上运行成功了

**Reply by TwooSix** [reply]
感谢，这个版本可以正常使用
